src/SFA.DAS.AssessorService.Application.Api.Client/SFA.DAS.AssessorService.Application.Api.Client.csproj

```diff
@@ -10,12 +10,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+	<FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Tab consistency

Reply:
These appear consistent on my local copy. All lines use spaces rather than tabs.

Reply:
They are not consistent here though so something is wrong and the problem will likely spread

Reply:
Appears as a tab when all others are spaces.